packages/perseus/src/widgets/radio/choice-icon/sat-choice-icon.jsx

```diff
@@ -87,8 +79,7 @@ function SATChoiceIcon(props: SATChoiceIconProps): React.Node {
                 style={{backgroundColor, borderColor, borderRadius}}
             />
             <div style={{color}} className={css(styles.letter)}>
-                <span className="perseus-sr-only">{a11yText}</span>
-                <span aria-hidden="true">{letter}</span>
+                <span>{letter}</span>
```

Comment:
This seems like a change in behavior and I wonder if it was intentional. Before the SAT choice would read "Choice A (unchecked) this is a choice" or whatever, now it looks like it will read "A Choice A (unchecked) this is a choice"...since the letter used to be hidden to screen readers.

If this is intentional, no worries. Did we check SAT with a screen reader?

Reply:
Because Clickable (the outer component for the button choice) is set to aria-hidden="true", it hides everything inside, which includes ChoiceIcon and OptionStatus. This information is provided to the screen reader via the a11y text now. I also manually tested the SAT via Storybook just now and it sounds as it should. Thank you for pointing this out though!